### DISCUSSION TITLE
[Feature Request]: 希望能加入下载器监控里面的“目录隔离”功能（目录隔离以外的下载任务不触发转移和刮削）

### DISCUSSION THREAD

**Discussion by 188334911** [jxxghp/MoviePilot#3468] [讨论]
### 当前程序版本

V1.77

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

希望能加入nt一样的下载器监控转移里面的目录隔离功能，（目录隔离以外的下载任务不触发转移和刮削），目录实时监控功能没有下载器监控好用
![image](https://github.com/jxxghp/MoviePilot/assets/54146909/882a2c63-1d5c-4370-92ff-b9afd914e37d)

### 参考资料

NT

**Comment by yesugao** [comment]
就是不触发的呀，，我自己拉的视频进去他都不会触发，你是不是开了目录隔离插件

**Comment by 188334911** [comment]
> 就是不触发的呀，，我自己拉的视频进去他都不会触发，你是不是开了目录隔离插件

嗯，我的意思是在mp的下载器监控里面加入这个目录隔离功能

![1711963526452](https://github.com/jxxghp/MoviePilot/assets/54146909/41e96216-16c6-41e9-8fc2-99aab4647f9e)
